src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -0,0 +1,151 @@
+            _httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+        }
+
+        public Task<JObject> GetAdminSitesByHostName(string stampName, string[] hostNames)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<JObject> GetAdminSitesBySiteName(string stampName, string siteName)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<IEnumerable<object>> GetAppServiceEnvironmentDeployments(string hostingEnvironmentName)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<JObject> GetAppServiceEnvironmentDetails(string hostingEnvironmentName)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<IEnumerable<Dictionary<string, string>>> GetCertificatesInResourceGroup(string subscriptionName, string resourceGroupName)
+        {
+            throw new NotImplementedException();
+        }
+
+        public async Task<Dictionary<string, List<RuntimeSitenameTimeRange>>> GetRuntimeSiteSlotMap(string siteName)
+        {
+            var response = await GetObserverResource($"sites/{siteName}/runtimesiteslotmap");
+            var slotTimeRangeCaseSensitiveDictionary = JsonConvert.DeserializeObject<Dictionary<string, List<RuntimeSitenameTimeRange>>>(response);
+            var slotTimeRange = new Dictionary<string, List<RuntimeSitenameTimeRange>>(slotTimeRangeCaseSensitiveDictionary, StringComparer.CurrentCultureIgnoreCase);
+            return slotTimeRange;
+        }
+
+        public async Task<Dictionary<string, List<RuntimeSitenameTimeRange>>> GetRuntimeSiteSlotMap(string stampName, string siteName)
+        {
+            var response = await GetObserverResource($"stamp/{stampName}/sites/{siteName}/runtimesiteslotmap");
+            var slotTimeRangeCaseSensitiveDictionary = JsonConvert.DeserializeObject<Dictionary<string, List<RuntimeSitenameTimeRange>>>(response);
+            var slotTimeRange = new Dictionary<string, List<RuntimeSitenameTimeRange>>(slotTimeRangeCaseSensitiveDictionary, StringComparer.CurrentCultureIgnoreCase);
+            return slotTimeRange;
+        }
+
+        public Task<IEnumerable<Dictionary<string, string>>> GetServerFarmsInResourceGroup(string subscriptionName, string resourceGroupName)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<string> GetServerFarmWebspaceName(string subscriptionId, string serverFarm)
+        {
+            throw new NotImplementedException();
+        }
+
+        public async Task<IEnumerable<string>> GetSiteHostNames(string siteName)
+        {
+            var response = await GetObserverResource($"sites/{siteName}/hostnames?api-version=2.0");
+            var hostnames = response.Remove(0, 1).Remove(response.Length - 2, 1).Split(new char[] { ',' });
```

Comment:
What are these removes doing?

Reply:
The response is like a JSON Array. I'm just removing the opening and closing brackets